pages/_app.tsx

```diff
@@ -1,4 +1,6 @@
 import '../styles/globals.scss';
+import 'react-loading-skeleton/dist/skeleton.css';
```

Comment:
please put external libs at the top.

I think we will need to install the eslint sort dependencie lib as it becomes messy.

https://eslint.org/docs/latest/rules/sort-imports